gradle.properties

```diff
@@ -25,4 +25,4 @@ systemProp.gradle.internal.testdistribution.writeTraceFile=true
 systemProp.gradle.internal.testdistribution.queryResponseTimeout=PT20S
 
 # Default performance baseline
-defaultPerformanceBaselines=8.1-commit-f6b887c2
+defaultPerformanceBaselines=85ebea1
```

Comment:
That doesn't look right. It should be something like `8.1-commit-...` or `8.2-commit-...`.

Moreover, the PR should always reference a performance issue why we are rebasing.

Reply:
- Docs are not correct in this case, I tried to follow it to the letter.
- Issue referenced from private

Reply:
We have some docs what the baseline can look like: https://bt-internal-docs.grdev.net/gbt/performance/how-performance-test-works/#baseline

Though the step-by-step instructions don't clarify what you need to do with the short commit id you obtain.

Reply:
Found it meanwhile, and I don't understand it.

> A commit version, e.g. 8.0-commit-6d0169890d7.

What is a commit version? :D

Reply:
This time, I haven't found another level of something, this needs to be elaborated.

Reply:
A commit version is a version of Gradle pointing to a specific commit. The format is `<base-version>-commit-<short commit hash>`.

Reply:
But where `base-version` comes from? It doesn't look like something coming from [releases.json](https://github.com/gradle/dotorg-site/blob/71757280cbc7f6bf40eed83c36a1d315f15e4786/_data/releases.json) nor [releases.xml](https://github.com/gradle/web/blob/34a35380aca7f9c51d1cfdfc2b2e4207932e0868/data/releases.xml).